src/com/javarush/task/task08/task0822/Solution.java

```diff
@@ -0,0 +1,61 @@
+package com.javarush.task.task08.task0822;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+/*
+Минимальное из N чисел
+*/
+
+public class Solution {
+    public static void main(String[] args) throws Exception {
+        List<Integer> integerList = getIntegerList();
+        System.out.println(getMinimum(integerList));
+    }
+
+    public static int getMinimum(List<Integer> array) {
+        // Найти минимум тут
+        int min = Integer.MAX_VALUE;
+        for (Integer number : array) {
+            if (number < min) {
+                min = number;
+            }
+        }
+        return min;
```

Comment:
Вариант решения для подумать
```suggestion
 if (array.isEmpty()) {
            return 0;
        } else {
            Collections.sort(array, Comparator.naturalOrder());
            return array.iterator().next();
        }
```